### ISSUE TITLE
Integrate academic references into Climate Maps page

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/arctic-eds#410]
Right now there's no link to the standard format academic reference for the Climate Maps page; stuff just kinda _is_ there.

One possibility is to have the active layer show the academic reference below it.  We could potentially extract out the academic references so we're not duplicating the HTML hither and thither.

**Comment by charparr** [comment]
Yeah, I'm not sure how to cram the references in there. We might consider just adding a bullet to the top that says something like "for academic references, load a report for a location you are interested in" but then again it wasn't long ago I was ready to cull the entire "Maps" section.

**Comment by brucecrevensten** [comment]
Closing, we will be spreading the maps throughout the app in a newer build so this isn't relevant anymore.
